instrumentation/servlet/servlet-javax-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/javax/JavaxCommonServletAccessor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.javax;
+
+import io.opentelemetry.instrumentation.servlet.javax.JavaxServletAccessor;
+import javax.servlet.http.HttpServletRequest;
+
+public class JavaxCommonServletAccessor extends JavaxServletAccessor<Void> {
```

Comment:
I found this class confusing, I think can delete with comment in `RequestDispatcherAdviceHelper`, or if you want to keep, add comment here explaining it's purpose (and I think throw `UnsupportedOperationException` in the methods below)

Reply:
Removed use of `JavaxServletAccessor` in `RequestDispatcherAdviceHelper` and deleted this.